__tests__/app.test.js

```diff
@@ -65,23 +65,78 @@ describe("app", () => {
                 expect(typeof res.body.topics).toBe("object")
                 expect(res.body.topics).toHaveLength(3)
             })
+        });
+    })
+
+    describe.only("GET /api/articles/:article_id/comments", () => {
+        test("should return an array of objects with expected properties", () => {
+            return request(app).get('/api/articles/1/comments').expect(200).then((res) => {
+                const result = res.body.comments
 
+                for (var key in result) {
+                    var obj = result[key];
+                    expect(obj).toMatchObject({
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        created_at: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        article_id: expect.any(Number)
+                    })
+                }
+            })
+        });
+
+        test("should return an array of objects in order", () => {
+            return request(app).get('/api/articles/1/comments').expect(200).then((res) => {
+                const result = res.body.comments
+
+                expect(result).toBeSortedBy('created_at', {
+                    descending: true,
+                    coerce: true,
+                });
+            })
```

Comment:
We have one more 'happy path' test missing here. It is possible and valid for an article to have no associated comments (you can find an example of this in your data set). In this case, I would expect a status of 200 and an empty array returned